WooCommerce/Classes/ViewRelated/Orders/OrdersTabbedViewController.swift

```diff
@@ -163,6 +169,18 @@ extension OrdersTabbedViewController {
         return button
     }
 
+    /// Create a `UIBarButtonItem` to be used as order creation trigger on the top-right.
+    ///
+    func createOrderCreationBarButtonItem() -> UIBarButtonItem {
```

Comment:
The name of the method seems a little bit redundant, since you are using `create` and `creation`. I'd say to rename it like 
```suggestion
    func createNewOrderBarButtonItem() -> UIBarButtonItem {
```

Reply:
Totally agree 😄
Fixed in f660b22